src/SprykerEco/Yves/Payone/Form/KlarnaSubForm.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Payone\Form;
+
+use Generated\Shared\Transfer\PaymentTransfer;
+use Generated\Shared\Transfer\PayoneKlarnaTransfer;
+use Spryker\Yves\StepEngine\Dependency\Form\SubFormInterface;
+use SprykerEco\Shared\Payone\PayoneConstants;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class KlarnaSubForm extends AbstractPayoneSubForm
+{
+    protected const PAYMENT_METHOD = 'klarna';
```

Comment:
* Could you please sort them, public fist then protected.

Reply:
++